### DISCUSSION TITLE
mingw build, debugging and rnote-cli on windows

### DISCUSSION THREAD

**Discussion by Doublonmousse** [flxzt/rnote#832] [Development]
I have a couple of questions :

- When compiling from source for windows, is there a fundamental difference between the .exe generated by mingw64 and the exe generated by inno (other than the fact that the first one is not portable and the second is) ?   
- How to debug rnote on windows (get the `log::trace` to show in the terminal, be in mingw or the windows terminal) ?
- Is `rnote-cli` present in the windows `.exe` version ?

**Comment by Doublonmousse** [comment]
I figured I could launch rnote with debug using `RUST_LOG=rnote=debug rnote` in the mingw64 terminal after compiling from source.

**Comment by flxzt** [comment]
I always use mingw's environment for debugging as well. Regarding the other questions: trace logs can be enabled via `RUST_LOG=rnote=trace`. The cli is a separate binary, so it's not included in `rnote.exe`. The installer currently does not include it.
